packages/config/config/devices/0x0312/templates/minoston_template.json

```
@@ -499,5 +499,67 @@
 		"maxValue": 20,
 		"defaultValue": 5,
 		"unsigned": true
-	}
+	},
+    "power_reporting": {
+		"label": "Power Reporting",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 1,
+		"defaultValue": 0,
+		"unsigned": true,
+        "allowManualEntry": false,
+		"options": [
+			{
+				"label": "Enabled",
+				"value": 0
+			},
+			{
+				"label": "Disabled",
+				"value": 1
+			}
+		]
+    },
+    "current_reporting": {
+        "label": "Current Reporting",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 1,
+		"defaultValue": 0,
+		"unsigned": true,
+        "allowManualEntry": false,
+		"options": [
+			{
+				"label": "Enabled",
+				"value": 0
+			},
+			{
+				"label": "Disabled",
+				"value": 1
+			}
+		]
+    },
```

```suggestion
	"power_reporting": {
		"$import": "~/templates/master_template.json#base_enable_disable_inverted",
		"label": "Power Reporting",
		"defaultValue": 0
	},
	"current_reporting": {
		"$import": "~/templates/master_template.json#base_enable_disable_inverted",
		"label": "Current Reporting",
		"defaultValue": 0
	},
```
We have templates to simplify these